tests/saving_data_test.py

```diff
@@ -18,15 +18,15 @@ def test_simple_saving() -> None:
     tests that a DataTuple can be saved
```

Comment:
```suggestion
    Tests that a DataTuple can be saved.
```

Reply:
I'm not committing that because then I'd have to wait for another 6 minutes.